pom.xml

```diff
@@ -92,8 +92,39 @@
 			<artifactId>aws-java-sdk-sts</artifactId>
 			<version>${amazonaws.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>io.rest-assured</groupId>
+			<artifactId>rest-assured</artifactId>
+			<version>5.3.1</version>
+			<exclusions>
+				<exclusion>
+					<groupId>org.apache.groovy</groupId>
+					<artifactId>groovy-xml</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+		<dependency>
+			<groupId>com.beust</groupId>
+			<artifactId>jcommander</artifactId>
+			<version>1.48</version>
```

Comment:
Create variables for version in properties tag instead of giving version here. Something like this.
<properties>
  <jcommander.version>1.48</jcommander.version>
</properties>

Do it for other dependencies as well which you have added.